R/install_greta_deps.R

```diff
@@ -142,6 +124,22 @@ install_greta_deps <- function(deps = greta_deps_spec(),
 
 }
 
+get_pkg_user_dir <- function() {
+  pkg_user_dir <- tools::R_user_dir("greta")
+  if (!dir.exists(pkg_user_dir)) {
+    dir.create(pkg_user_dir, recursive = TRUE)
+  }
+  pkg_user_dir
+}
+
+greta_default_logfile <- function(){
+  greta_user_dir <- get_pkg_user_dir()
+  default_greta_logfile_path <- glue::glue(
+    "{greta_user_dir}/greta-installation-logfile.html"
```

Comment:
why not `file.path(greta_user_dir, "greta-installation-logfile.html")`? I once read one should not paste to create a path but I can't remember why (the slash difference on OS? something else?)

Reply:
Good point! I'm in a habit using glue for everything.